### PR TITLE
Meme-bot blog

### PR DESCRIPTION
**Pull request by MrZilinXiao**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-07-18-python-wechaty-meme-bot.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+ title: "基于Python-wechaty建立一个斗图机器人"
+ date: 2020-07-19 14:00 +0800
+ author: MrZilinXiao
+ header:
+
```
Comment (lijiarui) [inline _posts/2020-07-18-python-wechaty-meme-bot.md:7]: Please add a teaser for this blog.

File: _posts/2020-07-18-python-wechaty-meme-bot.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+本项目 [基于Python-wechaty建立一个斗图机器人] 系 暑期2020 支持的开源项目。
+
+## [基于Python-wechaty建立一个斗图机器人]具体计划
+
+- 导师：黄纯洪
+- 学生：肖子霖
+- 模块列表
+- 项目初步规划图（可能有调整）
+![image](https://upyun.mrxiao.net/img/wechaty-meme-bot.png)
```
Comment (lijiarui) [inline _posts/2020-07-18-python-wechaty-meme-bot.md]: Please localize the image in case of a future 404 error.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-18-python-wechaty-meme-bot.md (hunk 1 above)
Comment (MrZilinXiao) [reply _posts/2020-07-18-python-wechaty-meme-bot.md:7]: Already done, sorry for the inconvenience.

File: _posts/2020-07-18-python-wechaty-meme-bot.md (hunk 2 above)
Comment (MrZilinXiao) [reply _posts/2020-07-18-python-wechaty-meme-bot.md]: Already go localized, sorry for the inconvenience.

File: _posts/2020-07-18-python-wechaty-meme-bot.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+ title: "基于Python-wechaty建立一个斗图机器人"
+ date: 2020-07-19 14:00 +0800
+ author: MrZilinXiao
```
Comment (lijiarui) [inline _posts/2020-07-18-python-wechaty-meme-bot.md]: Here should be  `godkillerxiao`, or the system cannot recognize your profile.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-18-python-wechaty-meme-bot.md (hunk 3 above)
Comment (MrZilinXiao) [reply _posts/2020-07-18-python-wechaty-meme-bot.md]: Changed.

File: _posts/2020-07-18-python-wechaty-meme-bot.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+ title: "基于Python-wechaty建立一个斗图机器人"
+ date: 2020-07-19 14:00 +0800
+ author: godkillerxiao
+ header:
+  teaser: /assets/2020/meme-bot/07-19-wechaty-meme-bot.png
```
Comment (lijiarui) [inline _posts/2020-07-18-python-wechaty-meme-bot.md:6]: There shouldn't have any blank here.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
And you should resolve the conflict.

File: _posts/2020-07-18-python-wechaty-meme-bot.md (hunk 4 above)
Comment (huan) [reply _posts/2020-07-18-python-wechaty-meme-bot.md:6]: I guess you mean `indention`?

Comment (lijiarui) [reply _posts/2020-07-18-python-wechaty-meme-bot.md:6]: yes ：）

**Comment by MrZilinXiao** [comment]
> And you should resolve the conflict.

Yep. Done!

File: _posts/2020-07-18-python-wechaty-meme-bot.md (hunk 4 above)
Comment (MrZilinXiao) [reply _posts/2020-07-18-python-wechaty-meme-bot.md:6]: No more unnecessary indentions!

**Review by lijiarui — APPROVED** [review]
Good job！

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Comment by MrZilinXiao** [comment]
> Congratulations! 🎉 🎉 🎉
> 
> You become a wechaty contributor successfully! 👏 👏 👏
> 
> Please:
> 
> * [ ]   Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
> * [ ]  Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
> * [ ]   Join wechaty org by clicking https://github.com/wechaty
> 
> image

Already finished the last two steps, while I am not invited to Contributor Room after sending you PR Link.

**Pull request merged**